Dockerfile

```diff
@@ -1,27 +1,5 @@
-# Build the manager binary
-FROM golang:1.17 as builder
-
-WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
-
-# Copy the go source
-COPY main.go main.go
-COPY api/ api/
-COPY controllers/ controllers/
-
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o manager main.go
-
-# Use distroless as minimal base image to package the manager binary
-# Refer to https://github.com/GoogleContainerTools/distroless for more details
-FROM gcr.io/distroless/static:nonroot
+FROM gcr.io/distroless/static:debug-nonroot
```

Comment:
Update from the debug tag that matched the rukpak container image to debug-nonroot as I ran into the following error locally:

```
container has runasnonroot and image will run as root
```

And that's because kubebuilder/operator-sdk automatically scaffold out that securityContext for our manager resources by default (and we didn't use that SDK tooling in rukpak).